src/System.Text.Json/src/System/Text/Json/Serialization/Converters/DefaultEnumConverter.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal class DefaultEnumConverter<TValue> : JsonValueConverter<TValue>
```

Comment:
All of these derived internal types that aren't derived from should be sealed.  I'll stop commenting on them.

Reply:
I'll mark them sealed. Thanks